packages/webpack/src/config/base.js

```diff
@@ -259,12 +262,12 @@ export default class WebpackBaseConfig {
           // item in transpile can be string or regex object
           return !this.modulesToTranspile.some(module => module.test(file))
         },
-        use: perfLoader.js().concat(babelLoader)
+        use: perfLoader.js().concat(this.getBabelLoader())
```

Comment:
Why we have to call `getBabelLoader` multiple times?

Reply:
Well in fact it in this method it's called 3 times, 2 being TypeScript related and will be removed soon.
So there will be one call here + one call later for the ability to have this babelLoader.

I resolve this as you suggested extracting babel loader config from config in another comment, let's talk there.